ui/client/reducers/graph/reducer.ts

```diff
@@ -322,7 +283,7 @@ const reducer = mergeReducers(
     processToDisplay: {
       nodes,
       properties: {
-        additionalFields: {groups},
+        additionalFields: {},
```

Comment:
is explicit setting additionalFields still required ?

Reply:
I'd leave it for next refactor....